src/Console/ConsoleOutput.php

```diff
@@ -309,6 +309,7 @@ public function outputAs($type = null)
             return $this->_outputAs;
         }
         $this->_outputAs = $type;
+        return null;
```

Comment:
yes, and I could disable the warning but I'm not bothered by the extra return. It shows the true intent of the code. It clarifies that the return value will be null and not void (as an example). :+1: